### ISSUE TITLE
Update visuals of the PF.org training pages

### ISSUE THREAD

**Issue by lboehling** [patternfly/patternfly-design#1236]
As a part of the website refresh, we identified that the visuals on the training pages do not follow PF styles. Want to refine the visuals on these pages to use PF styles. 

https://www.patternfly.org/v4/training/design
https://www.patternfly.org/v4/training/html
https://www.patternfly.org/v4/training/react
https://www.patternfly.org/v4/training/react-charts

**Comment by maryshak1996** [comment]
Design mocks: https://www.sketch.com/s/661ef9d9-8307-4e50-84e5-16eea2ff18d0
@evwilkin @mcarrano , Not sure what the next steps are here but I vaguely remember some POC talk??

**Comment by mcarrano** [comment]
@nicolethoen perhaps also take a look. Can't recall if this is something we are trying to address for v5 or a future initiative.

**Comment by nicolethoen** [comment]
I don't think there is a huge reason that we need to get this in for the v5 release, but if most of these updates are to the 'card' page which lays out the options for training (which is what it looks like to mo?) then it's a relatively small change. we could likely implement that rather quickly.

**Comment by mcarrano** [comment]
Should I just open an Org issue in that case and we can prioritize accordingly?

**Comment by evwilkin** [comment]
@mcarrano yes that'd be great, thanks.  Agreed that any enhancements to the website that don't impact functionality don't need to be prioritized before V5 release work.

@maryshak1996 the sketch link shows 2 different proposals, I'm not sure what the differences are but we'll need to note which changes we want to make.

**Comment by maryshak1996** [comment]
Hey @evwilkin ! Ya, so there was sort of two routes that we could take based on how much time/effort we are willing and able to put into this:

1. Design option 1 is a much smaller lift because it would have the design trainings link off to the external Thinkific platform that it currently does. The drawback here is that (a) users are required to created a Thinkific account to go through the training and (b) updates to the Thinkific training need to be done through an admin interface which just has one single login (mine), meaning that all updates are on me at this point.
3. Design option 2 is a larger lift because it uses primarily existing patternfly components (mostly the in page wizard) to build out a 'learning path' UI (very similar to what developers.redhat.com ([example](https://developers.redhat.com/learn/openshift/how-deploy-application-using-red-hat-openshift-service-aws)) and cloud.redhat.com ([example](https://cloud.redhat.com/learn/getting-started-red-hat-openshift-service-aws-rosa?extIdCarryOver=true&sc_cid=701f2000001Css5AAC)) do) that would live within the PattenFly site. This option, while more time consuming, is definitely a better experience for the user and is more maintainable long term because we wouldn't have a dependency on Thinkific.

**Comment by phcox** [comment]
Irrelevant at this time.
